src/testUtils/java/org/broadinstitute/hellbender/testutils/BaseTest.java

```diff
@@ -318,45 +318,6 @@ public static void log(final String message) {
 
     private static final double DEFAULT_FLOAT_TOLERANCE = 1e-1;
 
-
-    /**
-     * Checks whether two double array contain the same values or not.
-     * @param actual actual produced array.
-     * @param expected expected array.
-     * @param tolerance maximum difference between double value to be consider equivalent.
-     */
-    protected static void assertEqualsDoubleArray(final double[] actual, final double[] expected, final double tolerance) {
```

Comment:
I'm not sure I'm a fan of replacing this with a call to a TestNG method marked `internal` that doesn't seem like a good idea...

Reply:
Keeping the old method.